.github/workflows/daily-build.yml

```diff
@@ -0,0 +1,56 @@
+name: Daily build
+
+on:
+  workflow_dispatch:
+  repository_dispatch:
+    types:
+      check_connector_for_breaking_changes
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: 11
+
+      - name: Set environment variable
+        if: github.event.action == 'check_connector_for_breaking_changes'
+        run: |
+          echo "BUILD_USING_DOCKER=-PbuildUsingDocker=nightly" >> $GITHUB_ENV
+          echo "GRADLE_SKIP_TASKS=-x :mysql-compiler-plugin-tests:test"  >> $GITHUB_ENV
```

Comment:
```suggestion
          echo "GRADLE_SKIP_TASKS= " >> $GITHUB_ENV
```

Reply:
Fixed in [dd1e6ba](https://github.com/ballerina-platform/module-ballerina-persist/pull/1/commits/dd1e6bafacd813e06e6f9affaa56e6ff8b9ebd16)